Sources/SnapshotTesting/Strategy/UIViewController.swift

```diff
@@ -7,36 +7,21 @@ extension Strategy where Snapshottable == UIViewController, Format == UIImage {
   }
 
   public static func image(
-    drawingHierarchyInKeyWindow: Bool = false,
```

Comment:
Removing this option for now. Will try to add back soon but it was making the refactor a bit of a pain.

Reply:
(And not sure what to do about the fact that we need to use a custom window for safe area insets. Maybe you have ideas?)